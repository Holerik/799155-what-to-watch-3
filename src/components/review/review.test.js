// review.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import Review from './review.jsx';

const reviewInfo = [
  {
    id: 33,
    text: `Review text`,
    author: `review author`,
    date: `2016-12-20`,
    rating: `7,2`,
  }
];


it(`<Review /> should render some review`, () => {
  const tree = renderer
    .create(<Review
      review={reviewInfo[0]}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
