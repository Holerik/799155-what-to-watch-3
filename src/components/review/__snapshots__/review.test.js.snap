// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Review /> should render some review 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Review text
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        review author
      </cite>
      <time
        className="review__date"
        dateTime="2016-12-20"
      >
        December 20, 2016
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    7,2
  </div>
</div>
`;
